convert2rhel/checks.py

```diff
@@ -832,3 +844,88 @@ def check_dbus_is_running():
         "Could not find a running DBus Daemon which is needed to register with subscription manager.\n"
         "Please start dbus using `systemctl start dbus` or (on CentOS Linux 6), `service messagebus start`"
     )
+
+
+def _verify_initramfs_file(latest_installed_kernel):
+    """Internal function to handle the verification for initramfs file.
+
+    :param latest_installed_kernel: The version corresponding to the latest installed kernel on the system.
+    :type latest_installed_kernel: str
+    :return: A boolean to determinate if there was any errors with the file or not.
+    :rtype: bool
+    """
+    initramfs_file = INITRAMFS_FILEPATH % latest_installed_kernel
+
+    if not os.path.exists(initramfs_file):
+        return False
+
+    logger.info("Checking if the %s file is not corrupted.", initramfs_file)
+    _, err, return_code = run_piped_subprocess(
+        cmd1=["zcat", initramfs_file],
+        cmd2=["cpio", "-i", "--to-stdout"],
+        skip_stdout_decode=True,
+    )
```

Comment:
Did you want to redirect stdout output to /dev/null?

Reply:
Yes, I'm fine with the stdout for this to be redirected to `/dev/null` or `pipe`. For that, we only care about stderr and return_code if any, at least for this call.